### DISCUSSION TITLE
Critical vulnerability for rack 0.0.0 in docker-scout

### DISCUSSION THREAD

**Discussion by IainMcHugh** [pact-foundation/pact-node#495] [Q&A]
Hello

I am currently using `"@pact-foundation/pact": "^12.3.0"` as a dev dependency in a NextJS project. Docker-scout is listing a critical vulnerability for `rack 0.0.0` , and the package path is:
```
Type: gem
Location: 
  /node_modules/@pact-foundation/pact-core/standalone/darwin-arm64-2.4.2/pact/lib/vendor/ruby/3.2.0/gems/rack-2.2.8.1/rack.gemspec
  /node_modules/@pact-foundation/pact-core/standalone/darwin-x64-2.4.2/pact/lib/vendor/ruby/3.2.0/gems/rack-2.2.8.1/rack.gemspec
  /node_modules/@pact-foundation/pact-core/standalone/linux-arm64-2.4.2/pact/lib/vendor/ruby/3.2.0/gems/rack-2.2.8.1/rack.gemspec
  /node_modules/@pact-foundation/pact-core/standalone/linux-x64-2.4.2/pact/lib/vendor/ruby/3.2.0/gems/rack-2.2.8.1/rack.gemspec
  /node_modules/@pact-foundation/pact-core/standalone/windows-x64-2.4.2/pact/lib/vendor/ruby/3.2.0/gems/rack-2.2.8.1/rack.gemspec
```
We upgraded the `@pact-foundation/pact` version based on [this](https://github.com/pact-foundation/pact-ruby-standalone/issues/132) issue hoping it would fix this but the location is linking back to `@pact-foundation/pact-core`

**Comment by YOU54F** [comment]
Please see response on https://github.com/pact-foundation/pact-js-core/issues/501
